data-prepper-plugins/otel-proto-common/src/test/java/com/amazon/dataprepper/plugins/codec/OTelProtoCodecTest.java

```diff
@@ -0,0 +1,696 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.codec;
```

Comment:
I think including `otel` somehow in the package is appropriate.

I'd recommend either: `com.amazon.dataprepper.otel.codec` or `com.amazon.dataprepper.plugins.otel.codec`.

This isn't really a plugin, so that is why my first suggestion dropped the `plugin` from the package name.

Reply:
I will use the latter package name as for now it still belongs to plugins